dist/tools/firmware_metadata/README.md

```diff
@@ -0,0 +1,35 @@
+# Metadata generator for firmware verification
+This program will generate a binary file containing a metadata structure as
```

Comment:
Minor: use present instead of future.
`This program generates a binary file...`